src/components/Calender.vue

```diff
@@ -15,9 +15,10 @@
 		>
 			<template v-if="!calVal.isPadding">
 				<h3>
+					{{ calVal.date + 1 }}
 					{{
 						(filmExist(calVal.date)) ? 
-						"".concat(String(calVal.date + 1) + " " + listToDisplay.get(calVal.date)!.name: ''
+						" ".concat(listToDisplay.get(calVal.date)!.name : ''
```

Comment:
error is that there is no closing parenthesis on the concat. But also there's nothing concatenating here anymore, so not needed